src/ansible_development_environment/cli.py

```diff
@@ -36,6 +36,10 @@ def parse_args(self: Cli) -> None:
         self.args = parse()
         if hasattr(self.args, "requirement") and self.args.requirement:
             self.args.requirement = Path(self.args.requirement).expanduser().resolve()
+        if self.args.cpi:
+            self.args.requirement = (
+                Path(".config/source-requirements.yml").expanduser().resolve()
```

Comment:
Comment: Let's wait and see if anyone wants this to be configurable, having it hard coded isn't great but fine for now I think